colossalai/inference/server/api_server.py

```diff
@@ -0,0 +1,180 @@
+import argparse
+import json
+
+import uvicorn
+from fastapi import FastAPI, Request
+from fastapi.responses import JSONResponse, Response, StreamingResponse
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from colossalai.inference.config import InferenceConfig
+from colossalai.inference.server.completion_service import ColossalAICompletionServing
+from colossalai.inference.server.utils import id_generator
+
+from colossalai.inference.core.async_engine import AsyncInferenceEngine, InferenceEngine  # noqa
+
+TIMEOUT_KEEP_ALIVE = 5  # seconds.
+app = FastAPI()
```

Comment:
Give some docstrings on how to invoke this server from cmd somewhere in this file.

Reply:
Add a complete doc